Tests/SwiftDocCTests/LinkTargets/LinkDestinationSummaryTests.swift

```diff
@@ -276,7 +276,7 @@ class ExternalLinkableTests: XCTestCase {
             let summary = node.externallyLinkableElementSummaries(context: context, renderNode: renderNode)[0]
             
             XCTAssertEqual(summary.title, "globalFunction(_:considering:)")
-            XCTAssertEqual(summary.relativePresentationURL.absoluteString, "/documentation/mykit/globalfunction(_:considering:)")
+            XCTAssertEqual(summary.relativePresentationURL.absoluteString, "/documentation/mykit/globalfunction(__considering_)")
```

Comment:
I don't think this value should change. It represents the relative URL for this page. As far as I understood the URL would still contain the characters that aren't allowed in the file names.

Reply:
Right, the URLs can contain the view, but the file paths would be replaced.  I believe that the replativePresentationURL is still referencing the file path?

Reply:
No, this is meant to refer to the web URL relative to where this documentation is hosted. 

This API is used to bridge with other documentation systems which aren't expected to have access to the doccarchive directly. Those systems are expected to use the summary information to create links that navigate the reader to the hosted documentation.

Reply:
It can also refer to a landmark on a page which wouldn't have its own file in the documentation archive.